### DISCUSSION TITLE
Traveller Electro Talent Materials

### DISCUSSION THREAD

**Discussion by sllewji** [xanthics/genshin_grind_planner#15] [Ideas]
Great little site for keeping track of  - thanks for your time in putting it together.
Now the 2.0 has hit the streets, it would be really nice if we could use this to track Traveller's Electro Talent materials, as well as Anemo and Geo.

Thanks in advance!

**Comment by xanthics** [comment]
![image](https://user-images.githubusercontent.com/3137623/128698778-f7b95242-1b12-41a5-aa7d-11060c8362b6.png)
 They are already present, since the launch of 2.0.  All of the traveler's available elements are at the bottom of the list.
 
 
![image](https://user-images.githubusercontent.com/3137623/128698900-f558b54e-9ae5-44f9-8172-1c9a3e794a94.png)

**Comment by sllewji** [comment]
Thanks Jeremy - I guess I must be using a cached copy.
I'll try updating.

Sent from my OnePlus phone

________________________________
